package.json

```diff
@@ -19,11 +21,24 @@
     "next": "13.0.7",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "storybook": "^6.5.14",
```

Comment:
WE don't need to install storybook as dependency, please put it into devDependencies.